lib/payload.json

```
@@ -137,6 +198,55 @@
             }
           },
           "additionalProperties": false
+        },
+        "water": {
+          "type": "object",
+          "properties": {
+            "leak": {
+              "type": "boolean",
+              "description": "Status of water leaking (true/false)"
```

```suggestion
              "description": "Leak detected"
```

This aligns with "Motion detected" below